pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -29,32 +29,34 @@ import (
 
 // ValidateDefaultPreemptionArgs validates that DefaultPreemptionArgs are correct.
 func ValidateDefaultPreemptionArgs(args config.DefaultPreemptionArgs) error {
```

Comment:
Eventually, we will make this function accept a path #88093

In preparation for that do the following:

```golang
var path field.Path
if err := validateMinCandidateNodesPercentage(args.MinCandidateNodesPercentage, path.Child("minCandidateNodesPercentage")) {
```

Do the same for all the functions